package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "cypress": "^6.1.0"
+  },
+  "dependencies": {
+    "puppeteer": "^5.5.0"
+  }
+}
```

Comment:
If this is deps for Cypress, I'd vote for creating the Node package in `/cypress` rather than root project dir.